src/bp/common/utterance-parser.ts

```diff
@@ -1,7 +1,5 @@
 import _ from 'lodash'
```

Comment:
for some reason, I can't put this file in nlu/ directory as frontend won't build